templates/default/rt/rtticketeditbox.html

```diff
@@ -116,6 +116,19 @@
                                                                         </SELECT>
                                                                 </TD>
                                                         </TR>
+                                                        <TR>
+                                                                <TD style="width: 1%;" class="bold nobr">
+                                                                        <IMG src="img/calendar.gif" alt="">
+                                                                        {trans("Deadline time:")}
+                                                                </TD>
+                                                                <TD style="width: 99%;">
+									<INPUT TYPE="TEXT" NAME="ticket[deadlinetime]" class="calendar-time" VALUE="{$ticket.deadlinetime|date_format:"%Y/%m/%d %H:%M"}" SIZE="20" {tip text="Enter deadline time in YYYY/MM/DD HH:MM format or click to select it from applet" trigger="deadlinetime"}>
+                        {if $ticket.deadlinetimewarning}
+                        <INPUT TYPE="HIDDEN" NAME="ticket[deadlinetimewarning]" VALUE="1">
```

Comment:
Czemu to służy? Nie znalazłem, żeby to gdzieś indziej było używane.

Reply:
Poprawiłem. Warning zrobię w późniejszych commitach.